tests/__mocks__/adapter.ts

```diff
@@ -0,0 +1,62 @@
+import { Context } from "../../src/types";
+import { Comment } from "../../src/adapters/supabase/helpers/comment";
+import { Embedding } from "../../src/adapters/openai/helpers/embedding";
+import { STRINGS } from "./strings";
+
+export interface CommentMock {
+  id: number;
+  commentbody: string;
+  issuebody: string;
+  embedding: number[];
+}
+
+export function createMockAdapters(context: Context) {
+  const commentMap: Map<number, CommentMock> = new Map();
```

Comment:
I guess this would be reset every run. If we run twice the logic on the same group of comments, would this be problematic?

Reply:
Isn't that the expected behavior ? In case of collision it should throw error. I don't think there would be collision in comment id.

Reply:
During the run it is fine, but all the results are pushed to supabase correct? I just wondered the impact of having duplicate content in the database.

Reply:
While its running, the `issue_comment.created` will create a new comment in the db. All the results are pushed to the database assuming there would be no collision in the `comment id`. Comments may contain duplicate content, but they cannot have the same ID.